src/App.js

```diff
@@ -1,51 +1,79 @@
 // Display the cocktail to the screen based on user selection
 
-import { useEffect } from "react";
+// Stretch Goal #1
+// include cocktail ingredients & recipe along with name & photo
+
+// Stretch Goal #2
+// Styling
+
+// Modules
+import { useEffect, useState } from "react";
 import axios from "axios";
-import './App.css';
 
-function App() {
+// Components
+import DynamicDrink from "./Components/DynamicDrink";
+
+
+const App = () => {
+  const [drinkResult, setDrinkResult] = useState(null);
+  const [userSelection, setUserSelection] = useState("initial");
+  const [form, setForm] = useState(null);
+  
+  const handleClick = (e) => {
+    e.preventDefault();
+    // Update form state to userSelection value
+    setForm(userSelection);
+  }
+
   useEffect(() => {
     axios({
       url: "https://www.thecocktaildb.com/api/json/v1/1/search.php",
       method: "GET",
       dataResponse: "json",
       params: {
-        s: "Old Fashioned"
+        s: userSelection
       },
     }).then((res) => {
-      console.log(res.data.drinks);
+      setDrinkResult(res.data.drinks);
     })
-  }, [])
+  }, [form])
 
   return (
-    <div className="App">
-      <h1>Bartendr</h1>
-      <p>Select your mood to reveal what cocktail you should make!</p>
-      <form>
-        <select name="mood" id="mood">
-          <option value="initial" disabled selected>How are you feeling?</option>
-          <option value="">Excited</option>
-          <option value="">Content</option>
-          <option value="">Angry</option>
-          <option value="">Depressed</option>
-          <option value="">Nervous</option>
-          <option value="">Peaceful</option>
-          <option value="">Nostalgic</option>
-          <option value="">Romantic</option>
+    <>
+    <div className="wrapper">
+      <h1>Mixed Feelings</h1>
+      <form onSubmit={handleClick}>
+        <label htmlFor="mood">Select your mood to reveal what cocktail you should make</label>
+        <select name="mood" id="mood" onChange={(e) => {setUserSelection(e.target.value)}} value={userSelection}>
+          <option value="initial" disabled>How are you feeling?</option>
+          <option value="Margarita">Excited</option>
+          <option value="Aperol_Spritz">Content</option>
+          <option value="Dark_and_Stormy">Angry</option>
+          <option value="White_Russian">Depressed</option>
+          <option value="Dry_Martini">Nervous</option>
+          <option value="Mojito">Peaceful</option>
+          <option value="Old_Fashioned">Nostalgic</option>
+          <option value="Champagne_Cocktail">Romantic</option>
+          <option value="Pink_Lady">Whimsical</option>
+          <option value="Espresso_Martini">Lethargic</option>
+          <option value="Bloody_Mary">Vengeful</option>
+          <option value="Grasshopper">Jealous</option>
+          <option value="Long_Island_Iced_Tea">Confused</option>
         </select>
         <button>Submit</button>
       </form>
```

Comment:
Inventive and creative combos for the options! An interesting future stretch goal might be to allow users to add their own cocktail + moods into your list using firebase and another input form.